### PR TITLE
Move info from docs into README

### PR DESCRIPTION
**Pull request by crandmck**
@0candy 
Per https://github.com/strongloop/loopback.io/issues/158 I'm moving info from docs into the README, as for the other connectors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,176 +1,352 @@
 # loopback-connector-postgresql
 
-The official PostgreSQL connector for the LoopBack framework.
+[PostgreSQL](https://www.postgresql.org/), is a popular open-source object-relational database.
+The `loopback-connector-postgresql` module is the PostgreSQL connector for the LoopBack framework.
 
-Please see the [official documentation](http://loopback.io/doc/en/lb2/PostgreSQL-connector.html).
+<div class="gh-only">For more information, see the <a href="http://loopback.io/doc/en/lb2/PostgreSQL-connector.html">documentation</a>.
```
Comment (0candy) [inline README.md]: http://loopback.io/doc/en/lb3/PostgreSQL-connector.html

File: README.md [hunk 2]
```diff
@@ -1,176 +1,352 @@
...
-model:
+model: a table, for example, PRODUCT under the 'public' schema within the database.
+
+The auto-migrate method:
 
-* A table, for example, PRODUCT under the 'public' schema within the database
+* Defines a primary key for the properties whose `id` property is true (or a positive number).
+* Creates a column with 'SERIAL' type if the `generated` property of the `id` property is true.
 
+Destroying models may result in errors due to foreign key integrity. First delete any related models first calling delete on models with relationships.
```
Comment (0candy) [inline README.md]: First delete any related models ~~first~~ `by` calling delete on models with relationships.

File: README.md (hunk 1 above)
Comment (crandmck) [inline README.md]: Ack, cut and paste error propagation! :-( 
Fixed, thanks.

File: README.md (hunk 2 above)
Comment (crandmck) [inline README.md]: Fixed.

**Pull request merged**